### PR TITLE
Generate Jupyter Notebook w/o papermill

### PR DESCRIPTION
**Pull request by philipqueen**
This PR removes our papermill dependency by manually setting the recording folder path in the ipynb as a JSON object. The main downside is the jupyter notebook is no longer run automatically, although it is set up to run properly by clicking "Run All". Hopefully this removes some of the notebook based problems we've had, and simplifies the process for this step.

### CODE REVIEW

**Comment by jonmatthis** [comment]
I get this running on Windows 11


image

File: freemocap/data_layer/export_data/generate_jupyter_notebook/generate_jupyter_notebook.py [hunk 1]
```diff
@@ -13,14 +11,15 @@ def generate_jupyter_notebook(path_to_recording: Union[str, Path]):
...
-    logger.info(f"Jupyter notebook generated at {path_to_output_notebook}")
-    success = pm.execute_notebook(
-        input_path=path_to_template_notebook,
-        output_path=path_to_output_notebook,
-        parameters=dict(path_to_recording=str(path_to_recording)),
-    )
+    with open(path_to_template_notebook, "r") as file:
+        template_notebook = json.load(file)
+
+    template_notebook["cells"][3]["source"] = [f'path_to_recording = "{str(path_to_recording)}"']
```
Comment (jonmatthis) [inline freemocap/data_layer/export_data/generate_jupyter_notebook/generate_jupyter_notebook.py:17]: This is ok enough for now, but that hard coded `3` is a pretty brittle solution. We'll have to remember to update it when we change our ipynb template (which we should do soon!)

Comment (philipqueen) [reply freemocap/data_layer/export_data/generate_jupyter_notebook/generate_jupyter_notebook.py]: Unfortunately, `template_notebook["cells"]` is a list, so we don't have many options for indexing.

**Comment by philipqueen** [comment]
There we go, no persisting problem. Good to know the calculator graph issue is somewhat random though

**Comment by philipqueen** [comment]
Had some branch confusion and things got pushed to the wrong place, should be good now

**Review by jonmatthis — APPROVED** [review]

**Pull request merged**